src/contribute/release-guide.md

```diff
@@ -511,7 +512,171 @@ If there are no issues, reply on the vote thread to close the voting. Then, tall
+  -Prepourl=https://repository.apache.org/content/repositories/orgapachebeam-1041 \
+  -Pver= ${RELEASE_VERSION}\
+  -PgcpProject=${YOUR_GCP_PROJECT} \
+  -PgcsBucket=${YOUR_GCP_BUCKET}
+  ```
+* Java Mobile Game(UserScore, HourlyTeamScore, Leaderboard)
+  
+  Pre-request
+  * Create your own BigQuery dataset
+    ```
+    bq mk --project=${YOUR_GCP_PROJECT} ${YOUR_DATASET}
+    ```
+  * Create yout PubSub topic 
+    ```
+    gcloud alpha pubsub topics create --project=${YOUR_GCP_PROJECT} ${YOUR_PROJECT_PUBSUB_TOPIC} 
+    ```
+  * Setup your service account
+    
+    Goto IAM console in your project to create a service account as ```project owner```
+    
+    Run 
+    ```
+    gcloud iam service-accounts keys create ${YOUR_KEY_JSON} --iam-account ${YOUR_SERVICE_ACCOUNT_NAME}@${YOUR_PROJECT_NAME}
+    export GOOGLE_APPLICATION_CREDENTIALS=${PATH_TO_YOUR_KEY_JSON} 
+    ```
+  Run 
+  ```
+  ./gradlew :beam-runners-google-cloud-dataflow-java:runMobileGamingJavaDataflow \
+   -Prepourl=https://repository.apache.org/content/repositories/orgapachebeam-1041 \ 
+   -Pver= ${RELEASE_VERSION}\
+   -PgcpProject=${YOUR_GCP_PROJECT} \
+   -PgcsBucket=gs://dataflow-samples/game/gaming_data1.csv \
+   -PbqDataset=${YOUR_DATASET} -PpubsubTopic=${YOUR_PROJECT_PUBSUB_TOPIC}
+  ```
+* Python Quickstart(batch & streaming), MobileGame(UserScore, HourlyTeamScore)
+  
+  Create a new PR in apache/beam
+  
+  In comment area, type in ```Run Python ReleaseCandidate```
+ 
+* Python Leaderboard & GameStats
+  * Get staging RC ```wget https://dist.apache.org/repos/dist/dev/beam/2.5.0/* ```
+  * Verify the hashes
+  
+    ```
+    sha512sum -c apache-beam-2.5.0-python.zip.sha512
+    sha512sum -c apache-beam-2.5.0-source-release.zip.sha512
+    ```
+  * Build SDK
+  
+    ```
+    sudo apt-get install unzip
+    unzip apache-beam-2.5.0-source-release.zip
+    ```
+  * Setup virtualenv
+    
+    ```
+    pip install --upgrade pip
+    pip install --upgrade setuptools
+    pip install --upgrade virtualenv
+    virtualenv beam_env
+     . beam_env/bin/activate
+    ```
+  * Install SDK
+    
+    ```
+    pip install dist/apache-beam-2.5.0.tar.gz
+    pip install dist/apache-beam-2.5.0.tar.gz[gcp]
+    ```
+  * Setup GCP
+    
+    Please repeat following steps for every following test
+    
+    ```
+    bq rm -rf --project=${YOUR_PROJECT} ${USER}_test
+    bq mk --project=${YOUR_PROJECT} ${USER}_test
+    gsutil rm -rf ${YOUR_GS_STORAGE]
+    gsutil mb -p ${YOUR_PROJECT} ${YOUR_GS_STORAGE}
+    gcloud alpha pubsub topics create --project=${YOUR_PROJECT} ${YOUR_PUBSUB_TOPIC}
+    ```
+    Setup your service account as described in ```Java Mobile Game``` section above
+  
+    Produce data by using java injector
+    
+    ```
+    mvn archetype:generate \
+          -DarchetypeGroupId=org.apache.beam \
+          -DarchetypeArtifactId=beam-sdks-java-maven-archetypes-examples \
+          -DarchetypeVersion=2.4.0 \
```

Comment:
why 2.4.0 here?

Reply:
We just use java injector here, so we can get jar from last release.

Reply:
We should generate it from the release being validated, so 2.5.0 now, 2.6.0 for the next release

Reply:
It seems like, we don't upload artifacts into maven central  during vote process. So we cannot get release version artifacts using this cmd before it releases.

Reply:
You can get the artifacts from the staging repo, you need to configure an archetype repo in your `~/.m2/settings.xml` with an archetype repo like:
```
<settings>
  <profiles>
   <profile>
     <id>release-repo</id>
     <activation>
       <activeByDefault>true</activeByDefault>
     </activation>
     <repositories>
       <repository>
         <id>Release 2.4.0 RC3</id>
         <name>Release 2.4.0 RC3</name>
         <url>https://repository.apache.org/content/repositories/orgapachebeam-1031/</url>
       </repository>
     </repositories>
   </profile>
  </profiles>
</settings>
```

and specify `-DarchetypeCatalog=internal`

Reply:
I added more instructions in commit https://github.com/apache/beam-site/pull/468/commits/9b0ec38866cdd67a2eba415ae318f3872011e723. @lukecwik  @aaltay